client-side-virtual-keyboard.md

```diff
@@ -0,0 +1,229 @@
+# Client Side Virtual Keyboard
+
+Anbox Streaming SDK enables developers to build a hybrid mobile application that can integrate rich features that Anbox Cloud provides. It comes with an [Android library](https://developer.android.com/studio/projects/android-library) which offers easy-to-use native components like AnboxWebView which helps to quickly create client side virtual keyboard feature integrated mobile client that can send the text to Android container on the fly when typing. It extends the AOSP [WebView](https://developer.android.com/reference/android/webkit/WebView) and provides better handling of the text input for the hybrid application that loads the Anbox Streaming JavaScript SDK with an embedded WebView for video streaming.
```

Comment:
That looks good to me.
I prefer to use
`...which you can use to quickly integrate a client-side virtual keyboard feature into your mobile client..`
over
`...which you can use to quickly integrate a client-side virtual keyboard feature into your mobile application..` to avoid double `client` terms in one sentence.
How do you feel about that?

Reply:
Yes, that sounds better. Thanks!